arbnode/seq_coordinator.go

```diff
@@ -896,6 +898,30 @@ func (c *SeqCoordinator) Start(ctxIn context.Context) {
 	if c.config.ChosenHealthcheckAddr != "" {
 		c.StopWaiter.LaunchThread(c.launchHealthcheckServer)
 	}
+	// Verifies that c.config.MyUrl is valid and is in the priorities list in redis
+	priorities, err := c.RedisCoordinator().GetPriorities(ctxIn)
```

Comment:
skip the test id MyUrl is either empty or INVALID_URL (some units have a sequencer for read access only)